scripts/format-code.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env sh
+
+BFT="concord-bft"
+
+if [ -z "$1" ]; then
+  >&2 echo "ERROR: Path to ${BFT} directory required"
```

Comment:
I wonder if we could be smarter here, and just try "." or ".." if no path is given. It might also be nice to add the usual example style to the end, e.g. `Usage: format-code.sh <concord-bft path>`.

Reply:
I think I prefer the Usage example. We don't want people to accidentally format everything and get spooked. Then again, we really do just want to run this from the top-level dir. I'll add usage.

Reply:
Fixed.